crystal/shivneri/shard.yml

```diff
@@ -0,0 +1,17 @@
+name: app
+version: 0.1.0
+
+authors:
+  - Ujjwal Gupta <ujjwalkumargupta44@gmail.com>
+
+targets:
+  app:
```

Comment:
could you replace the target list with
~~~yaml
targets:
  server:
    main: src/server.cr
~~~

Reply:
done, also changed host to 0.0.0.0